pkg/elevate/elevate.go

```diff
@@ -64,10 +64,10 @@ func RunElevate(argv []string) error {
 
 	elevateCmd := "oc " + strings.Join(argv[1:], " ")
 
-	shell := "/bin/bash"
-
-	if len(argv) > 3 {
-		shell = argv[4]
+	// Make the Default SHELL environment variable to /bin/bash
+	shell := os.Getenv("SHELL")
+	if shell == "" {
```

Comment:
What if throws an error for the empty shell, or we need to check the validity of `/bin/bash` shell before making it as default shell.